src/settings_internal.js

```diff
@@ -28,6 +28,9 @@ var SYSCALLS_REQUIRE_FILESYSTEM = 1;
 // the features section.
 var BINARYEN_FEATURES = [];
 
+// Tracks whether to enable Wasm SIMD support in Binaryen.
+var BINARYEN_SIMD = 0;
```

Comment:
Why is this needed? I think it should work automatically with `BINARYEN_FEATURES` like other wasm features. Maybe I'm forgetting something @tlively ?

Reply:
It does not seem to. Not sure if it's not hooked up, or?

Reply:
That could be a bug in LLVM then if it isn't writing out the simd feature @tlively ?

Reply:
Yeah this should Just Work™️. I can look into it as a follow-up.